crates/bevy_window/src/window.rs

```diff
@@ -534,422 +580,121 @@ impl Window {
-    pub fn set_decorations(&mut self, decorations: bool) {
-        self.decorations = decorations;
-        self.command_queue
-            .push(WindowCommand::SetDecorations { decorations });
-    }
-    /// Get whether or how the cursor is grabbed.
-    ///
-    /// ## Platform-specific
-    ///
-    /// - **`Windows`** doesn't support [`CursorGrabMode::Locked`]
-    /// - **`macOS`** doesn't support [`CursorGrabMode::Confined`]
-    /// - **`iOS/Android`** don't have cursors.
-    ///
-    /// Since `Windows` and `macOS` have different [`CursorGrabMode`] support, it's possible the value returned here is not the same as the one actually sent to winit.
-    #[inline]
-    pub fn cursor_grab_mode(&self) -> CursorGrabMode {
-        self.cursor_grab_mode
-    }
-    /// Set whether and how the cursor is grabbed.
-    ///
-    /// This doesn't hide the cursor. For that, use [`set_cursor_visibility`](Window::set_cursor_visibility)
-    ///
-    /// ## Platform-specific
-    ///
-    /// - **`Windows`** doesn't support [`CursorGrabMode::Locked`]
-    /// - **`macOS`** doesn't support [`CursorGrabMode::Confined`]
-    /// - **`iOS/Android`** don't have cursors.
-    ///
-    /// Since `Windows` and `macOS` have different [`CursorGrabMode`] support, we first try to set the grab mode that was asked for. If it doesn't work then use the alternate grab mode.
-    pub fn set_cursor_grab_mode(&mut self, grab_mode: CursorGrabMode) {
-        self.cursor_grab_mode = grab_mode;
-        self.command_queue
-            .push(WindowCommand::SetCursorGrabMode { grab_mode });
-    }
-    /// Get whether or not the cursor is visible.
-    ///
-    /// ## Platform-specific
-    ///
-    /// - **`Windows`**, **`X11`**, and **`Wayland`**: The cursor is hidden only when inside the window. To stop the cursor from leaving the window, use [`set_cursor_grab_mode`](Window::set_cursor_grab_mode).
-    /// - **`macOS`**: The cursor is hidden only when the window is focused.
-    /// - **`iOS`** and **`Android`** do not have cursors
-    #[inline]
-    pub fn cursor_visible(&self) -> bool {
-        self.cursor_visible
-    }
-    /// Set whether or not the cursor is visible.
-    ///
-    /// ## Platform-specific
-    ///
-    /// - **`Windows`**, **`X11`**, and **`Wayland`**: The cursor is hidden only when inside the window. To stop the cursor from leaving the window, use [`set_cursor_grab_mode`](Window::set_cursor_grab_mode).
-    /// - **`macOS`**: The cursor is hidden only when the window is focused.
-    /// - **`iOS`** and **`Android`** do not have cursors
-    pub fn set_cursor_visibility(&mut self, visible_mode: bool) {
-        self.cursor_visible = visible_mode;
-        self.command_queue.push(WindowCommand::SetCursorVisibility {
-            visible: visible_mode,
-        });
-    }
-    /// Get the current [`CursorIcon`]
+    /// You probably don't want to call this directly unless you are dealing
+    /// with a window manager library.
```

Comment:
Ya this was meant as in it would be updated by winit and company, but this isn't entirely true. I'm going to reword this to just state that it just ignores the scale factor, which you *probably* don't want to do.